src/includes/configuration/config-intro/flutter.mdx

```diff
@@ -1,11 +1,11 @@
-Options are passed to the `SentryClient()` class:
+Options are passed to the `Sentry.init` method:
 
 ```dart
-final sentry = SentryClient(
-  dsn: '___PUBLIC_DSN___',
-  environmentAttributes: const Event(
-    release: 'my-project-name@2.3.12',
-    environment: 'production',
-  )
-);
+  void options(SentryOptions options) {
+    options.dsn = '___PUBLIC_DSN___';
+    options.release = 'my-project-name@2.3.12';
+    options.environment = 'production';
```

Comment:
Maybe we keep `production` out since it's default?

```suggestion
    options.environment = 'staging';
```

Reply:
makes sense, we should change in java too